### ISSUE TITLE
Unable to use Keyboard shortcuts in the explorer since 1.76.0

### ISSUE THREAD

**Issue by Bilboramix** [microsoft/vscode#176858]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.76.1
- OS Version: Windows_NT x64 10.0.22621

Steps to Reproduce:

1. Try to rename a file in the explorer with a keyboard shortcut (F2 by default)
2. Try to delete a file in the explorer with a keyboard shortcut (Del by default)

"Reveal in the file explorer" with shift + alt + r is the only one that still works for me

**Comment by ulugbekna** [comment]
Hi @Bilboramix ,

Thanks for filing the issue! 

Could you, please, have a look at this keybindings troubleshooting [guide](https://github.com/microsoft/vscode/wiki/Keybinding-Issues#basic-troubleshooting)? Following the steps of "Basic troubleshooting" should help you find out why the keybinding is not working. We could also maybe help if you paste the output here.

**Comment by eiskalteschatten** [comment]
This is also applies to copying and pasting files with `ctrl+c` (or `cmd+c`) and `ctrl+v` (or `cmd+v`) in the File Explorer. It no longer works even though my keybindings are set correctly and those are the shortcuts shown in the context menu. Using the context menu with the mouse still works.

**Comment by lramos15** [comment]
Have you moved your explorer somewhere? Does `Reset View Locations` in the command palette fix it? This has also been fixed in 1.77 insiders in. case you want to try that out https://code.visualstudio.com/insiders/

**Comment by eiskalteschatten** [comment]
I haven't moved the Explorer anywhere else. It's in its default position on the left side of the screen (unless you count accidentally dragging it into the Source Control view (https://github.com/microsoft/vscode/issues/176384).

I couldn't reproduce the problem with copying and pasting files in the insiders version 1.77. Maybe the issue has already been addressed?

**Comment by lramos15** [comment]
Yes I've fixed this in 1.77.0 unfortunately that fix cannot be brought into the stable version. There's a bug where moving the explorer can mess up the keybindings, even if it's moved back to the default location. You can try `Reset View Locations` to see if that will fix it.

**Comment by eiskalteschatten** [comment]
It looks like I ran into exactly that bug. Running `Reset View Locations` worked and fixed the problem for me. It fixed the original problem as well for me, but the original poster should probably try it as well...

**Comment by Bilboramix** [comment]
Hello, sorry for late response, i investigate about the bug an ```Reset View Locations``` did the job.
But i wanted to make it reproductible then i started to make non sense thing with the drag feature.
So i figured out that drag the entire "explorer" tab does not trigger the bug.
Moving the "folder" accordion _outside_ the explorer triggers the bug.

If you move back the accordion to the "explorer" tab, the bug is fixed.

Here's a gif to illustrate :

![Animation](https://user-images.githubusercontent.com/91626615/225112568-45b7a83f-1681-4ffe-8ce9-56fe0786cecd.gif)

Here's my output when the bug is active :

For ```CTRL + C```
```
2023-03-14 20:23:32.847 [info] [KeybindingService]: / Received  keydown event - modifiers: [ctrl], code: ControlLeft, keyCode: 17, key: Control
2023-03-14 20:23:32.847 [info] [KeybindingService]: | Converted keydown event - modifiers: [ctrl], code: ControlLeft, keyCode: 5 ('Ctrl')
2023-03-14 20:23:32.848 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-03-14 20:23:33.064 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-03-14 20:23:33.064 [info] [KeybindingService]: | Resolving ctrl+C
2023-03-14 20:23:33.064 [info] [KeybindingService]: \ From 7 keybinding entries, matched editor.action.clipboardCopyAction, when: no when condition, source: built-in.
2023-03-14 20:23:33.064 [info] [KeybindingService]: / Received  keydown event - modifiers: [ctrl], code: KeyC, keyCode: 67, key: c
2023-03-14 20:23:33.065 [info] [KeybindingService]: | Converted keydown event - modifiers: [ctrl], code: KeyC, keyCode: 33 ('C')
2023-03-14 20:23:33.065 [info] [KeybindingService]: | Resolving ctrl+C
2023-03-14 20:23:33.065 [info] [KeybindingService]: \ From 7 keybinding entries, matched editor.action.clipboardCopyAction, when: no when condition, source: built-in.
2023-03-14 20:23:33.065 [info] [KeybindingService]: + Invoking command editor.action.clipboardCopyAction.
2023-03-14 20:23:33.184 [info] [KeybindingService]: + Storing single modifier for possible chord ctrl.
2023-03-14 20:23:33.214 [info] [KeybindingService]: + Clearing single modifier due to other key up.
```

for ```Del```
```
2023-03-14 20:25:11.977 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-03-14 20:25:11.977 [info] [KeybindingService]: | Resolving Delete
2023-03-14 20:25:11.977 [info] [KeybindingService]: \ From 11 keybinding entries, no when clauses matched the context.
2023-03-14 20:25:11.977 [info] [KeybindingService]: / Received  keydown event - modifiers: [], code: Delete, keyCode: 46, key: Delete
2023-03-14 20:25:11.977 [info] [KeybindingService]: | Converted keydown event - modifiers: [], code: Delete, keyCode: 20 ('Delete')
2023-03-14 20:25:11.978 [info] [KeybindingService]: | Resolving Delete
2023-03-14 20:25:11.978 [info] [KeybindingService]: \ From 11 keybinding entries, no when clauses matched the context.
```

for ```Shift + &``` that is a custom keyboard shortcut i made to create a file (It works with the bug activated)
```
2023-03-14 20:27:23.091 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-03-14 20:27:23.091 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-03-14 20:27:23.091 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-03-14 20:27:23.307 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-03-14 20:27:23.307 [info] [KeybindingService]: | Resolving shift+1
2023-03-14 20:27:23.307 [info] [KeybindingService]: \ From 1 keybinding entries, matched explorer.newFile, when: no when condition, source: user.
2023-03-14 20:27:23.307 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: Digit1, keyCode: 49, key: 1
2023-03-14 20:27:23.308 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: Digit1, keyCode: 22 ('1')
2023-03-14 20:27:23.308 [info] [KeybindingService]: | Resolving shift+1
2023-03-14 20:27:23.308 [info] [KeybindingService]: \ From 1 keybinding entries, matched explorer.newFile, when: no when condition, source: user.
2023-03-14 20:27:23.308 [info] [KeybindingService]: + Invoking command explorer.newFile.
2023-03-14 20:27:23.421 [info] [KeybindingService]: + Storing single modifier for possible chord shift.
2023-03-14 20:27:23.457 [info] [KeybindingService]: + Clearing single modifier due to other key up.
```

for ```Alt + R + shift``` that is the only native shortcut that works with the bug activated
```
2023-03-14 20:30:29.773 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-03-14 20:30:29.773 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-03-14 20:30:29.773 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-03-14 20:30:29.780 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift,alt], code: AltLeft, keyCode: 18, key: Alt
2023-03-14 20:30:29.780 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift,alt], code: AltLeft, keyCode: 6 ('Alt')
2023-03-14 20:30:29.780 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-03-14 20:30:29.983 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-03-14 20:30:29.983 [info] [KeybindingService]: | Resolving shift+alt+R
2023-03-14 20:30:29.983 [info] [KeybindingService]: \ From 1 keybinding entries, matched revealFileInOS, when: !editorFocus, source: built-in.
2023-03-14 20:30:29.983 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift,alt], code: KeyR, keyCode: 82, key: R
2023-03-14 20:30:29.983 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift,alt], code: KeyR, keyCode: 48 ('R')
2023-03-14 20:30:29.984 [info] [KeybindingService]: | Resolving shift+alt+R
2023-03-14 20:30:29.984 [info] [KeybindingService]: \ From 1 keybinding entries, matched revealFileInOS, when: !editorFocus, source: built-in.
2023-03-14 20:30:29.984 [info] [KeybindingService]: + Invoking command revealFileInOS.
```

**Comment by lramos15** [comment]
@Bilboramix Thanks for the investigation, but this bug is already fixed in 1.77.0 insiders so if you do want to move your explorer I recommend trying out the insiders version!
